### DISCUSSION TITLE
Add support for SWIFT Wallet (ERC-4337) wallet creation

### DISCUSSION THREAD

**Discussion by ronaldoguedess** [trustwallet/wallet-core#3700] [Q&A]
Will you extend wallet creation to support the new standard ERC-4337 ?
or SWIFT Wallet

**Comment by satoshiotomakan** [comment]
Hi @ronaldoguedess, we do support ERC-4337, please take a look at the examples:
https://github.com/trustwallet/wallet-core/blob/master/tests/chains/Ethereum/BarzTests.cpp
